zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -109,4 +140,6 @@ service RemoteInterpreterService {
   RemoteApplicationResult loadApplication(1: string applicationInstanceId, 2: string packageInfo, 3: string noteId, 4: string paragraphId);
   RemoteApplicationResult unloadApplication(1: string applicationInstanceId);
   RemoteApplicationResult runApplication(1: string applicationInstanceId);
+
+  void remoteZeppelinServerControlFeedback(1: RemoteZeppelinServerController response);
```

Comment:
Please try use primitive types instead of `RemoteZeppelinServerController`.  You can json serialize complex data into string like other rpc methods above.
And more specific name, such as 'sendRunContexts()`